zerver/lib/digest.py

```diff
@@ -268,5 +268,5 @@ def handle_digest_email(user_profile_id: int, cutoff: float,
         # Send now, as a ScheduledEmail
         send_future_email('zerver/emails/digest', user_profile.realm, to_user_ids=[user_profile.id],
                           from_name="Zulip Digest", from_address=FromAddress.NOREPLY,
-                          context=context)
+                          language=user_profile.realm.default_language, context=context)
```

Comment:
I think we don't need this, since we have `to_user_ids`, right?

Reply:
It's only with `to_emails` where we need this.  And also, if there is a user object, we don't want to do the realm.default_language -- that's the default language for new users, not the current user's configured language (which they could have changed from that).